parquet-column/src/main/java/parquet/example/data/Group.java

```diff
@@ -72,6 +73,8 @@ public Group getGroup(String field, int index) {
     return getGroup(getType().getFieldIndex(field), index);
   }
 
+  abstract public  void add(int fieldIndex, Primitive value);
+
```

Comment:
How is this change related to "Add support for dictionaries to FilteringPrimitiveConverter" ?